### PR TITLE
Jonathan farrow117 patch 2

### PR DESCRIPTION
**Pull request by JonathanFarrow117**
## Description
Added Multitrack recording to NCCO section and recording description 

## Deploy Notes

### CODE REVIEW

File: _documentation/voice/voice-api/guides/recording.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ When recording a named conversation, the recording is always asynchronous and ti
 
 When recording a call, you can enable split recording which will result in the recording being a stereo file with one channel having the audio sent from the caller and another channel being the audio heard by the caller.
 
+## Multi channel recording
+
+When recording a call, you can enable mutli channel recording which allows up to 32 call legs to be recorded separately. One file with the number of channels set will be returned.
```
Comment (tbedford) [inline _documentation/voice/voice-api/guides/recording.md]: "mutli channel" -> typo. multichannel.

File: _documentation/voice/voice-api/ncco-reference.md [hunk 2]
```diff
@@ -59,7 +59,8 @@ You can use the following options to control a `record` action:
 Option | Description | Required
  -- | -- | --
 `format` | Record the Call in a specific format.  Options are: <ul><li>`mp3`</li><li>`wav`</li></ul> The default value is `mp3`. | No
-`split` | Record the sent and received audio in separate channels of a stereo recording—set to `conversation` to enable this.| No
+`split` | Record the sent and received audio in separate channels of a stereo recording—set to `conversation` to enable this.|No
+`channel` | Record up to `32` seprate channels. split `conversation` must also be enabled.| No
```
Comment (tbedford) [inline _documentation/voice/voice-api/ncco-reference.md]: Typo. 'sperate' -> separate.

File: _documentation/voice/voice-api/guides/recording.md (hunk 1 above)
Comment (JonathanFarrow117) [reply _documentation/voice/voice-api/guides/recording.md]: Fixed

File: _documentation/voice/voice-api/guides/recording.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ When recording a named conversation, the recording is always asynchronous and ti
 
 When recording a call, you can enable split recording which will result in the recording being a stereo file with one channel having the audio sent from the caller and another channel being the audio heard by the caller.
 
+## Multi channel recording
+
+When recording a call, you can enable mutlichannel recording which allows up to 32 call legs to be recorded separately. One file with the number of channels set will be returned.
```
Comment (tbedford) [inline _documentation/voice/voice-api/guides/recording.md]: Typo. "mutlichannel" -> multichannel.

**Pull request merged**